src/components/FormPluginEditor/M3RichTextEditor.module.scss

```
@@ -441,19 +441,45 @@ $m3-state-active: rgba(39, 50, 112, 0.12);
+// Owner call 2026-07-29: the glyph carries this control, not a button shell —
+// so the 36px pill loses its padding and the icon grows to 22px, which gives
+// the text next to it real width back. 24px is the floor, not padding: it is
+// the minimum pointer target, and at 22px the icon all but fills it.
 .toolBtn.maximizeToolBtn {
-    width: 36px;
-    height: 36px;
+    width: 28px;
+    height: 28px;
     flex-shrink: 0;
     border-radius: 50%;
+    // Active primary (owner call 2026-07-30): the bare glyph at 22px read as
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)